snowdrop-core/src/Snowdrop/Core/ERoComp/Types.hs

```diff
@@ -51,7 +51,7 @@ type family Ctx conf :: *
 --
 -- Type variables @id@, @value@ describe types of key and value of key-value storage.
 -- Variable @res@ denotes result of 'DbAccess' execution.
-data DbAccess (components :: [*]) (res :: *)
+data DbAccess conf (components :: [*]) (res :: *)
```

Comment:
why this phantom type is needed here?
it's absolutely messy to populate all datatypes in framework by all available type-parameters